packages/carbon-web-components/src/components/text-input/text-input-skeleton.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { prefix } from '../../globals/settings';
+import styles from './input.scss';
+
+/**
+ * Skeleton of number input.
+ */
+@customElement(`${prefix}-text-input-skeleton`)
+class CDSTextInputSkeleton extends LitElement {
+  /**
+   * `true` if the label should be hidden. Corresponds to the attribute with the same name.
+   */
```

Comment:
```suggestion
  /**
   * Specify whether the label should be hidden, or not
   */
```

We should match these with the same desc from `@carbon/react`